pkg/xero/auth.go

```diff
@@ -126,7 +126,7 @@ func exchangeToken(ctx context.Context, httpClient *http.Client, data *url.Value
 		if body == "" {
 			body = "no error body"
 		}
-		return "", "", status.Error(codes.Code(rawResponse.StatusCode), fmt.Sprintf("Request failed: %s", body))
+		return "", "", status.Error(codes.Code(rawResponse.StatusCode), fmt.Sprintf("Request failed: %s", body)) //nolint:gosec // disable G115
```

Comment:
style: if this used uhttp I think it takes care of this kind of partial body / error messages